basic/src/test/java/org/jboss/hal/testsuite/test/configuration/webservices/WebServicesTestCaseAbstract.java

```diff
@@ -78,4 +62,11 @@ protected void createHandlerChainHandler(String handlerChainName, String handler
                 .saveAndDismissReloadRequiredWindowWithState()
                 .assertWindowClosed();
     }
+
+    protected void createConfiguration(String configurationName) {
```

Comment:
Methods with similar name are used across testsuite to create resources in model. Please change its name to make it clear, that this method creates resource in GUI.